src/pages/Homepage/HomepageStatictics.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<div class="small-10 small-offset-1 large-10 large-offset-1 columns stats-box">
+		<div class="row">
+			<!-- loans funded -->
+			<div class="small-6 large-2 large-offset-1 columns stats-box__group even">
+				<div class="stats-box__stat">
+					${{ amountFunded | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					loans funded
+				</div>
+			</div>
+			<!-- # lenders  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numLenders | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					lenders
+				</div>
+			</div>
+			<!-- repayment rate -->
+			<div class="small-6 large-2 columns stats-box__group even">
+				<div class="stats-box__stat">
+					{{ repaymentRate | numeral('0%') }}
+				</div>
+				<div class="stats-box__stat-label">
+					repayment rate
+				</div>
+			</div>
+			<!-- # countries  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numCountries }}
+				</div>
+				<div class="stats-box__stat-label">
+					countries
+				</div>
+			</div>
+			<!-- # years -->
+			<div class="small-12 large-2 columns">
+				<div class="stats-box__stat">
+					{{ yearsSinceStartCalc }}
+				</div>
+				<div class="stats-box__stat-label">
+					years
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import { differenceInYears } from 'date-fns';
+import homepageStatistics from '@/graphql/query/homepage/homepageStatistics.graphql';
+
+export default {
+	data() {
+		return {
+			amountFunded: 1.4,
+			numLenders: 1.9,
+			repaymentRate: 0.96,
+			numCountries: 77,
+		};
+	},
+	inject: ['apollo'],
+	apollo: {
+		query: homepageStatistics,
+		preFetch: true,
+		result({ data }) {
+			this.amountFunded = _get(data, 'general.kivaStats.amountFunded');
+			this.numLenders = _get(data, 'general.kivaStats.numLenders');
+			this.repaymentRate = _get(data, 'general.kivaStats.repaymentRate') / 100;
+			this.numCountries = _get(data, 'general.kivaStats.numCountries');
+		},
+	},
+	computed: {
+		yearsSinceStartCalc() {
+			// Kiva was formally registered as a
+			// non-profit on October 12, 2005
+			const foundedDate = new Date(2005, 10, 12);
+			return differenceInYears(Date.now(), foundedDate);
+		},
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.stats-box {
+	border-radius: rem-calc(25);
+	background-color: $kiva-accent-blue;
+	color: $white;
+	padding: 3.5rem 0;
+
+	&__group {
```

Comment:
BEMin it up, woo! :)

Reply:
I've enjoyed working in BEM. It structures the code in a way that is quickly readable, I like it. 📖